streams/src/test/java/org/apache/kafka/streams/processor/internals/metrics/StreamsMetricsImplTest.java

```diff
@@ -147,7 +142,7 @@ public class StreamsMetricsImplTest {
     private final StreamsMetricsImpl streamsMetrics = new StreamsMetricsImpl(metrics, CLIENT_ID, VERSION, time);
 
     private static MetricConfig eqMetricConfig(final MetricConfig metricConfig) {
-        EasyMock.reportMatcher(new IArgumentMatcher() {
+        Mockito.argThat(new ArgumentMatcher<Object>() {
```

Comment:
```suggestion
        return Mockito.argThat(new ArgumentMatcher<MetricConfig>() {
```

With this change you can remove the type check on line 150.